e2e/core/pages/metadata-view.page.ts

```diff
@@ -107,28 +106,16 @@ export class MetadataViewPage {
         await BrowserVisibility.waitUntilElementIsNotVisible(this.editIcon);
     }
 
-    async editIconClick(): Promise<void> {
-        await BrowserActions.clickExecuteScript('button[data-automation-id="meta-data-card-toggle-edit"]');
-    }
-
-    async informationButtonIsDisplayed(): Promise<void> {
-        await BrowserVisibility.waitUntilElementIsClickable(this.informationButton);
-    }
-
-    async informationButtonIsNotDisplayed(): Promise<void> {
-        await BrowserVisibility.waitUntilElementIsNotVisible(this.informationButton);
+    async generalEditIconDisplayed(): Promise<void> {
```

Comment:
functions names should start with verb, please rename

Reply:
@Yasa-Nataliya name of that function is still not proper. You renamed it to editGeneralIconDisplayed. Better name would be: 'isEditGeneralIconDisplayed' or something like that. Now it starts with 'edit' verb what suggests that this function is editing something